includes/polling/mempools/fortios.inc.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+// FortiOS Mempool
+echo 'FortiOS Mempool';
+
+$mempool['total'] = (snmp_get($device, 'FORTINET-FORTIMANAGER-FORTIANALYZER-MIB::fmSysMemCapacity.0', '-OvQ') * 1024);
+$mempool['used']  = (snmp_get($device, 'FORTINET-FORTIMANAGER-FORTIANALYZER-MIB::fmSysMemUsed.0', '-OvQ') * 1024);
+$mempool['perc'] = (($mempool['used'] / $mempool['total']) * 100);
+$mempool['free']  = ($mempool['total'] - $mempool['used']);
+
+echo '(U: '.$mempool['used'].' T: '.$mempool['total'].' F: '.$mempool['free'].') ';
```

Comment:
Either remove this line or change it to use `d_echo()` so it only appears in debug.